packages/muelu/test/unit_tests/Adapters/AmgxOperatorAdapter.cpp

```diff
@@ -91,9 +91,12 @@ namespace MueLuTests {
 
       RCP<const Teuchos::Comm<int> > comm = TestHelpers::Parameters::getDefaultComm();
       int nx;
-      //different problem sizes chosen based upon number of MPI ranks
-      if(comm->getSize() == 4) nx = 200;
-      if(comm->getSize() == 1) nx = 91;
+      //disable amgx test in parallel
+      if(comm->getSize() > 1) 
```

Comment:
Does the CMakeLists.txt file for this test only run it with one process?  If so, it might be better just to let the test fail if users give it multiple processes.  That would prevent some other developer changing the process count in the CMakeLists.txt file, then thinking that the test passed when it actually just returned early.

Reply:
This is not a separate binary, this is part of the mega MueLu_UnitTests binary.

We can't turn this off at the cmake level...